checkov/common/output/report.py

```diff
@@ -72,7 +73,34 @@ def get_dict(self, is_quiet=False) -> dict:
                 "summary": self.get_summary()
             }
 
-    def get_exit_code(self, soft_fail: bool) -> int:
+    def get_exit_code(self, soft_fail: bool, soft_fail_on: Union[list, None] = None,
+                      hard_fail_on: Union[list, None] = None) -> int:
```

Comment:
@kartikp10 `Union` is for multiple types, if it's either a list or None, use the `Optional` operator, i.e. `Optional[list]`

Reply:
Thanks for pointing that out!